plugins/logitech-tap/fu-logitech-tap-device.c

```diff
@@ -30,7 +30,7 @@ fu_logitech_tap_device_init(FuLogitechTapDevice *self)
 	fu_device_add_flag(FU_DEVICE(self), FWUPD_DEVICE_FLAG_SIGNED_PAYLOAD);
 	fu_device_register_private_flag(FU_DEVICE(self),
 					FU_LOGITECH_TAP_HDMI_DEVICE_FLAG_NEEDS_REBOOT,
```

Comment:
maybe rename this FU_LOGITECH_TAP_HDMI_DEVICE_FLAG_SENSOR_NEEDS_REBOOT

Reply:
Done: https://github.com/fwupd/fwupd/pull/7270